ui/pages/notification-details/notification-details.tsx

```diff
@@ -59,7 +61,13 @@ function useEffectOnNotificationView(notificationData?: Notification) {
         },
       ]);
     }
-  }, [markNotificationAsRead, notificationData]);
+
+    return () => {
+      if (notificationData?.type === TRIGGER_TYPES.SNAP) {
+        setNotificationTimeout(notificationData.id);
+      }
+    };
+  }, []);
```

Comment:
Empty dep array is fine here. I will investigate our notifications on why we needed our effect for marking as read (I think we can remove it, but will do so in another PR)